packages/tab-scroller/src/mwc-tab-scroller.ts

```diff
@@ -54,7 +54,6 @@ export class TabScroller extends BaseElement {
   @query('.mdc-tab-scroller__scroll-content')
   protected scrollContentElement!: HTMLElement;
 
-  @eventOptions({passive: true})
```

Comment:
`EventListenerOptions` doesn't contain `passive` and hasn't for some time, so i'm unsure why this suddenly doesn't compile. Maybe inside lit-element, we should be using `AddEventListenerOptions` instead ([see here](https://github.com/Microsoft/TypeScript/blob/28f8fdaccd78de926722e115157da32a50b77f4a/lib/lib.dom.d.ts#L33)).

This line shouldn't be removed but passive isn't valid so maybe a PR to lit needs doing first?

Reply:
will be fixed by polymer/lit-element#326